.babelrc

```diff
@@ -0,0 +1,12 @@
+{
+    "presets": [
+        ["@babel/env", {
+            "targets": {
+                "browsers": ["last 2 versions"],
+                "node": "current"
+            },
+            "loose": true
+        }],
```

Comment:
if you want object rest/spread and class properties you should include those as well because they aren't in env (unless using shippedProposals option) and not in typescript preset because it only strips types

Reply:
@hzoo nice! I'll add these in. thanks for catching that